plinth/modules/dynamicdns/dynamicdns.py

```diff
@@ -0,0 +1,394 @@
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from django import forms
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.core import validators
+from django.core.urlresolvers import reverse_lazy
+from django.template.response import TemplateResponse
+from gettext import gettext as _
+import logging
+
+from plinth import actions
+from plinth import cfg
+from plinth import package
+
+LOGGER = logging.getLogger(__name__)
+EMPTYSTRING = 'none'
+
+subsubmenu = [{'url': reverse_lazy('dynamicdns:index'),
+               'text': _('About')},
+              {'url': reverse_lazy('dynamicdns:configure'),
+               'text': _('Configure')},
+              {'url': reverse_lazy('dynamicdns:statuspage'),
+               'text': _('Status')}
+              ]
+
+
+def init():
+    """Initialize the dynamicdns module"""
+    menu = cfg.main_menu.get('apps:index')
+    menu.add_urlname('Dynamic DNS', 'glyphicon-refresh',
+                     'dynamicdns:index', 40)
+
+
+@login_required
+@package.required('ez-ipupdate')
+def index(request):
+    """Serve dynamic DNS  page"""
+
+    index_subsubmenu = subsubmenu
+
+    return TemplateResponse(request, 'dynamicdns.html',
+                            {'title': _('dynamicdns'),
+                             'subsubmenu': index_subsubmenu})
```

Comment:
couldn't you use `subsubmenu` directly here and omit 
`index_subsubmenu = subsubmenu`?